collections/ansible_collections/cloudkit/service/roles/vm_settings/defaults/main.yml

```diff
@@ -0,0 +1,16 @@
+---
+vm_settings_template_default: 
+  cloud_init_config:
+    users:
+      - name: osac
+        sudo: "ALL=(ALL) NOPASSWD:ALL"
+        groups:
+          - wheel
+          - sudo
+        shell: /bin/bash
+        lock_passwd: true
+        ssh_authorized_keys: "{{ vm_settings_template_parameters.ssh_public_key | default([]) }}"
```

Comment:
I am not requesting a change here because I don't know enough about kubevirt, but typically cloud environments provide ssh keys via a metadata service (which can also be provided when using image-based configuration), and the individual distributions decide which user to configure with those credentials.

If kubevirt provides us with a mechanism for setting metadata, rather than creating a static username, that would be my preference.

To be clear: I think this is very low priority.

Reply:
@jkary If you haven't already, please open an issue to track this change.